infra/docker/depot/files/nginx/git.conf

```diff
@@ -0,0 +1,43 @@
+
+server {
+    listen [::]:80;
+    listen 0.0.0.0:80;
+    server_name git.example.com;
+
```

Comment:
Also include here;

```
include /var/www-allow/client1-allow.conf;
deny all;
```

Where `client1-allow.conf` is a list of the whitelisted ips